README.md

```diff
@@ -6,4 +6,4 @@
 
 Run the below command in your terminal. 
 
-`npm triple`
```

Comment:
Not a typo, i'm using a tool named npx.
https://www.npmjs.com/package/npx

Reply:
Shouldn't it be np**x** and triple**j** though?